plugins/Kaleidoscope-FocusSerial/src/kaleidoscope/plugin/FocusSerial.h

```diff
@@ -26,6 +26,15 @@
 #include "kaleidoscope/event_handler_result.h"  // for EventHandlerResult, EventHandlerResult::OK
 #include "kaleidoscope/key_defs.h"              // for Key
 #include "kaleidoscope/plugin.h"                // for Plugin
+// -----------------------------------------------------------------------------
+// Deprecation warning messages
+#include "kaleidoscope_internal/deprecations.h"  // for DEPRECATED
+
+#define _DEPRECATED_MESSAGE_FOCUS_HANDLEHELP                      \
+  "The `Focus.handeHelp()` method is deprecated. Please use\n"    \
```

Comment:
typo: should probably be `Focus.handleHelp()`

Reply:
Fixed, thanks!